soc/wch/ch32v/qingke_v4f/vector.S

```diff
@@ -18,9 +18,11 @@ GTEXT(__initialize)
 
 SECTION_FUNC(vectors, ivt)
 	.option norvc
+    #ifndef SOC_CH32V307
```

Comment:
If you remove these two lines, you have to change the value written into mtvec (to 0x7 in this case).